doc/user/metrics/README.md

```diff
@@ -74,15 +74,129 @@ In Kubernetes clusters where [OwnerReferencesPermissionEnforcement][ownerref-per
   - update
 ...
 ```
-
 ### Custom resource specific metrics
 
 By default operator will expose info metrics based on the number of the current instances of an operator's custom resources in the cluster. It leverages [kube-state-metrics][ksm] as a library to generate those metrics. Metrics initialization lives in the `cmd/manager/main.go` file of the operator in the `serveCRMetrics` function. Its arguments are a custom resource's group, version, and kind to generate the metrics. The metrics are served on `0.0.0.0:8686/metrics` by default. To modify the exposed metrics port number, change the `operatorMetricsPort` variable at the top of the `cmd/manager/main.go` file in the generated operator.
 
+### Custom Metrics using Operator-SDK
+To implement custom metric collection using the `metrics package` provided by SDK, follow the steps outlined below:
+1. Update the Operator: Update the operator to provide sufficient permisisons so that the required resources for prometheus can be created.
```

Comment:
What permission is required that it not added by default?

Reply:
I have removed this part, as the required permissions are added by default.